### PR TITLE
Update README file with recent maven coordinates

### PR DESCRIPTION
**Pull request by Dichotomia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,9 +101,9 @@ com.mebigfatguy.fb-contrib, and the artifact ID is fb-contrib. Eg:
 
 ~~~~
 <plugin>
-    <groupId>com.github.hazendaz.spotbugs</groupId>
+    <groupId>com.github.spotbugs</groupId>
```
Comment (ThrawnCA) [inline README.md:106]: This coordinate also appears on line 75.

**Review by ThrawnCA — CHANGES_REQUESTED** [review]
The Spotbugs Maven Plugin should use the `sb-contrib` artifact rather than `fb-contrib`.

File: README.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ For example, If you are using FindBugs with Eclipse (and you extracted Eclipse t
 ## Usage - Maven
 
 To include the fb-contrib detectors when checking your project with FindBugs/Spotbugs, you can use the [FindBugs Maven plugin](https://gleclaire.github.io/findbugs-maven-plugin/usage.html).
-Or [SpotBugs Maven plugin](https://www.github.com/hazendaz/spotbugs-maven-plugin) which is a fork of findbugs maven plugin to provide spotbugs integration.  The group ID for fb-contrib is
+Or [SpotBugs Maven plugin](https://github.com/spotbugs/spotbugs-maven-plugin) which is a fork of findbugs maven plugin to provide spotbugs integration.  The group ID for fb-contrib is
 com.mebigfatguy.fb-contrib, and the artifact ID is fb-contrib. Eg:
```
Comment (ThrawnCA) [inline README.md]: Starting from version 7.4.5, the SpotBugs Maven Plugin should use the `sb-contrib` artifact ID instead of `fb-contrib`.

**Review by ThrawnCA — CHANGES_REQUESTED** [review]

**Comment by mebigfatguy** [comment]
thanks

**Pull request merged**